app/src/main/java/org/gdg/frisbee/android/common/GdgActivity.java

```diff
@@ -327,7 +312,7 @@ protected void recordStartPageView(Action viewAction) {
     protected ResultCallback<Status> appIndexApiCallback(final String label) {
         return new ResultCallback<Status>() {
             @Override
-            public void onResult(Status status) {
+            public void onResult(@NonNull Status status) {
```

Comment:
See https://github.com/gdg-x/frisbee/pull/560/files#r55929152 for the same question.